rust/headless-client/src/imp_linux.rs

```diff
@@ -240,8 +240,8 @@ impl Callbacks for CallbackHandlerIpc {
         None
     }
 
-    fn on_update_resources(&self, resources: Vec<ResourceDescription>) {
-        tracing::info!(len = resources.len(), "New resource list");
+    fn on_update_resources(&self, resources: Vec<callbacks::ResourceDescription>) {
+        tracing::info!(?resources, "New resource list");
```

Comment:
This can't be logged as INFO because it has domain names and IPs and stuff, right? That's why I only had the length before in some places in the Clients.

Reply:
I should have caught these too. We need to keep logs like this to debug because otherwise it'll just consume too much disk space for all users.

Reply:
maybe it's a bit too often but I don't think there's any sensitive info here. I think it's safe to log users' resources

Reply:
This will be called pretty often now, so I'll leave at `debug` with only the length, otherwise I think it will be too spammy.